docs/sources/tempo/operations/backend_search.md

```diff
@@ -141,7 +203,197 @@ querier:
     [external_hedge_requests_up_to: <int>]
 ```
 
-See here for cloud-specific details:
+For cloud-specific details:
 
 - [AWS Lambda]({{< relref "./serverless_aws" >}})
 - [Google Cloud Run]({{< relref "./serverless_gcp" >}})
+
+## Settings that are safe to increase without major impact.
+
+Scaling up queriers is a safe way to add more query capacity.
+At Grafana Labs, we prefer to scale queries horizontally by adding more replicas.
+If you see out of memory (OOM) errors, it might be worth scaling the queriers vertically.
+
+There should always be two replicas of query-frontends in a cluster.
```

Comment:
I think we should change this to at least two replicas.

I would rewrite this to be `As of today, we recommend running at least two replicas of query-frontend`, and recommend scaling them vertically instead of horizontally.`

Reply:
How does this look? 
```suggestion
We recommend running at least two replicates of query-frontend. These replicas should be scaled vertically instead of horizontally. ```